### PR TITLE
Python Program to find Unique number .

### PR DESCRIPTION
**Pull request by narayan2111**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Unique_Number_III
Given an array that contains all elements occurring 3 times, but one occurs only once. Find that unique element.

### Checklist

- [Y] I've read the contribution guidelines.
- [Y] I've checked the issue list before deciding what to submit.
- [Y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#988

### CODE REVIEW

**Comment by narayan2111** [comment]
Kindly review this PR @plazzy99 @iamrajiv @atarax665 @Kajol-Kumari @Abhijit2505

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@narayan2111 Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

**Comment by narayan2111** [comment]
> @narayan2111 Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

Done Sir @iamrajiv

**Review by iamrajiv — APPROVED** [review]

File: Python/other/Unique_number_III.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Unique_Number_III
+# Given an array that contains all elements occurring 3 times, but one
+# occurs only once. Find that unique element.
+
+
+def uniquee_element(array, size):
+    for i in range(size):
+        j = 0
+        while(j < size):
```
Comment (Kajol-Kumari) [inline Python/other/Unique_number_III.py:9]: so I can see a `while` loop under `for` loop then how is the time complexity `O(size)` here?

Comment (narayan2111) [reply Python/other/Unique_number_III.py:9]: > so I can see a `while` loop under `for` loop then how is the time complexity `O(size)` here?

Sorry ma'am ,time complexity will be O(size*size),because while loop is under for loop

**Review by atarax665 — APPROVED** [review]
LGTM :+1:

File: Python/other/Unique_number_III.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        if (j == size):
+            return array[i]
+
+    return -1
+
+
+print("Enter Elements:")
+array = input().split()
+size = len(array)
+print("Uniuqe element:")
```
Comment (Kajol-Kumari) [inline Python/other/Unique_number_III.py]: Can we please correct the typo here? `Uniuqe`  to `Unique`

File: Python/other/Unique_number_III.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+print(uniquee_element(array, size))
+
+'''
+Sample I/O:
+    Input:
+    Enter Elements:
+    1 1 1 4 2 2 2 3 3 3
+
+    Output:
+    Uniuqe element:
```
Comment (Kajol-Kumari) [inline Python/other/Unique_number_III.py]: same here. Update it accordingly.

**Review by Kajol-Kumari — APPROVED** [review]
@narayan2111 Please make the minor changes requested, Rest LGTM :+1:

**Comment by narayan2111** [comment]
Updated the code, kindly merge this PR @Kajol-Kumari ma'am

**Comment by narayan2111** [comment]
Kindly merge this PR @iamrajiv

File: Python/README.md [hunk 4]
```diff
@@ -129,6 +129,10 @@ _add list here_
 - [Merging Sorted arrays](other/merge_sorted_arrays.py)
 - [Reverse a string using stack](other/Reverse_string_using_stack.py)
 - [Russian Peasant Algorithm](other/Russian_Peasant_Algorithm.py)
+- [Satellite Image Analysis using numpy](other/Satellite_Image_Analysis_using_numpy.ipynb)
+- [Swap Two Nibbles](other/Swap_two_nibbles.py)
+- [Unique Number III](other/Unique_number_III.py)
```
Comment (atarax665) [inline Python/README.md:144]: Only add link of your file here. Remove unnecessary changes.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/README.md (hunk 4 above)
Comment (narayan2111) [reply Python/README.md:144]: Done Sir @atarax665

**Comment by narayan2111** [comment]
Kindly review this PR @atarax665

**Comment by narayan2111** [comment]
Kindly review this PR @atarax665

**Comment by narayan2111** [comment]
Kindly review this PR @atarax665

**Comment by narayan2111** [comment]
Kindly review this PR  @plazzy99

**Comment by narayan2111** [comment]
Kindly review this PR @Abhijit2505

**Comment by narayan2111** [comment]
Kindly review this PR @atarax665 @plazzy99

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻
